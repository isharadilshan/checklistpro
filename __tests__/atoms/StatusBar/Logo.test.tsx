import React from 'react';
import {describe, expect} from '@jest/globals';
import {shallow, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

import StatusBar from '../../../src/components/atoms/StatusBar';

configure({adapter: new Adapter()});

const props = {
  translucent: true,
  background: 'blue',
};

describe('when rendering', () => {
  const wrapper = shallow(<StatusBar {...props} />);

  it('should render a component matching the snapshot', () => {
    const tree = toJson(wrapper);
    expect(tree).toMatchSnapshot();
    expect(wrapper).toHaveLength(1);
  });
});
