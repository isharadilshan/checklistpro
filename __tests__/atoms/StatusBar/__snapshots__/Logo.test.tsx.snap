// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when rendering should render a component matching the snapshot 1`] = `
<StatusBar
  backgroundColor="blue"
  barStyle="dark-content"
  translucent={true}
/>
`;
